src/state/trajectory/logics.ts

```diff
@@ -106,9 +107,12 @@ const requestPlotDataLogic = createLogic({
     type: REQUEST_PLOT_DATA,
 });
 
-const handleFileLoadError = (error: FrontEndError) => {
-    const dispatch = useDispatch();
-
+const handleFileLoadError = (
+    error: FrontEndError,
+    dispatch: <T extends ArgumentAction<string, undefined, undefined>>(
+        action: T
+    ) => T
```

Comment:
If someone has a better idea for how to type `dispatch` let me know. I just copied and pasted what TS was telling me to be the type of `dispatch` in the logic functions that call this function.

Reply:
it looks odd to me but passing `dispatch` in does seem to be the pattern that the parent function uses too.   If that type def makes ts linting and typecheck happy, then I'm ok with it.